test_scripts/RC/OnRCStatus/012_notification_by_revoking_module_by_policy.lua

```diff
@@ -51,6 +51,7 @@ end
 --[[ Scenario ]]
 runner.Title("Preconditions")
 runner.Step("Clean environment", common.preconditions, { true, 1 })
+runner.Step("Update SDL config", common.setSDLIniParameter, { "ApplicationListUpdateTimeout", 4000 })
```

Comment:
Could you please explain why this change is necessary. This test passes for me even without these changes

Reply:
@ShobhitAd Sure.
SDL has a timeout defined by `ApplicationListUpdateTimeout` configuration option.
Once new app is registered SDL will wait during this timeout if other applications are registered as well.
Once this timeout expires SDL will send `BC.UpdateAppList` request to HMI and starts PTU sequence.
In this way it's possible to receive a PT update for a set of applications at once and not to have a few PTU sequences for each app.
By default this timeout is 3000 ms. and it may be not enough for our case:
 - Registering of 1st app takes 2000ms
 - Activation of 1st app - 500ms
 - Allocation of a new module - 100ms

We have a window of 300-400ms to register 2nd app and not to have a separate PTU sequence for it.
So we just give SDL more time to wait for the 2nd app in order to have one PTU for both apps.